common/djangoapps/util/milestones_helpers.py

```diff
@@ -36,18 +30,18 @@ def add_prerequisite_course(course_key, prerequisite_course_key):
     and it would set newly created milestone as fulfilment
     milestone for course referred by `prerequisite_course_key`.
     """
-    if settings.FEATURES.get('MILESTONES_APP', False):
-        # create a milestone
-        milestone = add_milestone({
+    if settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES', False):
+        from milestones import api as milestones_api
```

Comment:
Nit: This might be clearer written as "fail fast". That is, if `ENABLED_PREREQUISITE_COURSES` is `False`, immediately return.